src/components/FadeInOutAnimation.tsx

```diff
@@ -94,7 +95,12 @@ export const FadeInOutAnimation: React.FC<Props> = (props: Props) => {
       <CSSTransition classNames={{...transitionClassNames}} appear in={transitionIn} timeout={timeout} onEntered={handleEntered} onExited={handleExited}>
         <FadeCover style={{background: props.background ? props.background : theme.palette.primary.main}}/>
       </CSSTransition>
-      <StyledText className={props.textClasses}>{props.noSpace ? '' : '\u00A0'}{props.textSecondary[textIndex]}</StyledText>
+      <StyledText 
+        className={props.textClasses} 
+        style={{color: props.multiColor ? Object.values(colors)[textIndex] : 'unset'}}
```

Comment:
@krisbitney I was having some trouble figuring out how to make this boolean type work properly. *I think* it should also probably should be refactored so that the multiColor is not in the fadeInOutAnimation, but I wasn't sure about that. I wanted to at least implement something so that it was working.